Packs/CommonTypes/Layouts/layoutscontainer-registryKey.json

```diff
@@ -7,306 +7,683 @@
+						"y": 1
+					},
+					{
+						"displayType": "ROW",
+						"h": 1,
+						"hideName": false,
+						"i": "main-fafa73b0-a9af-11ea-a4b1-a5bbd1788c5a",
+						"items": [
+							{
+								"args": {},
+								"endCol": 1,
+								"fieldId": "",
+								"height": 44,
+								"id": "fe326d30-a9af-11ea-a4b1-a5bbd1788c5a",
+								"index": 0,
+								"name": "Enrich indicator",
+								"scriptId": "Builtin|||enrichIndicators",
+								"sectionItemType": "button",
+								"startCol": 0
+							},
+							{
+								"args": {},
+								"dropEffect": "move",
+								"endCol": 2,
+								"fieldId": "",
+								"height": 44,
+								"id": "64422e90-bf71-11ea-b085-3f354e53d3e6",
+								"index": 0,
+								"listId": "main-fafa73b0-a9af-11ea-a4b1-a5bbd1788c5a",
+								"name": "Expire indicator",
+								"scriptId": "Builtin|||expireIndicators",
+								"sectionItemType": "button",
+								"startCol": 1
+							}
+						],
+						"maxW": 3,
+						"minH": 1,
+						"minW": 1,
+						"moved": false,
+						"name": "Actions",
+						"static": false,
+						"w": 1,
+						"x": 2,
+						"y": 0
+					},
+					{
+						"displayType": "CARD",
+						"h": 1,
+						"hideName": false,
+						"i": "main-41542550-b6de-11ea-824d-5b821ee9d562",
+						"items": [
+							{
+								"endCol": 1,
+								"fieldId": "assigneduser",
+								"height": 53,
+								"id": "4b120ad0-b6de-11ea-824d-5b821ee9d562",
+								"index": 0,
+								"sectionItemType": "field",
+								"startCol": 0
+							},
+							{
+								"dropEffect": "move",
+								"endCol": 2,
+								"fieldId": "assignedrole",
+								"height": 53,
+								"id": "4d3efcf0-b6de-11ea-824d-5b821ee9d562",
+								"index": 1,
+								"listId": "main-41542550-b6de-11ea-824d-5b821ee9d562",
+								"sectionItemType": "field",
+								"startCol": 1
+							}
+						],
+						"maxW": 3,
+						"minH": 1,
+						"minW": 1,
+						"moved": false,
+						"name": "Assignee",
+						"static": false,
+						"w": 1,
+						"x": 2,
+						"y": 3
+					}
+				],
+				"type": "custom"
+			},
+			{
+				"hidden": false,
+				"id": "kopixekjqw",
+				"name": "Additional Details",
+				"sections": [
+					{
+						"displayType": "ROW",
+						"h": 9,
+						"i": "kopixekjqw-24f18460-beb4-11ea-b519-21bdfb6f5a43",
+						"items": null,
+						"maxW": 3,
+						"minH": 1,
+						"minW": 1,
+						"moved": false,
+						"name": "Extended Details",
+						"static": false,
+						"type": "extendedDetails",
+						"w": 2,
+						"x": 0,
+						"y": 0
+					},
+					{
+						"displayType": "ROW",
+						"h": 9,
+						"i": "kopixekjqw-27fffce0-beb4-11ea-b519-21bdfb6f5a43",
+						"items": null,
+						"maxW": 3,
+						"minH": 1,
+						"minW": 1,
+						"moved": false,
+						"name": "Custom Details",
+						"static": false,
+						"type": "customDetails",
+						"w": 1,
+						"x": 2,
+						"y": 0
+					}
+				],
+				"type": "custom"
+			}
+		]
+    },
+    "indicatorsQuickView": {
+		"tabs": [
```

Comment:
fix indent here please

Reply:
![image](https://user-images.githubusercontent.com/45915502/88475393-f643a280-cf37-11ea-932a-bea43a325e7b.png)
Looks good in my env.